pkg/util/log/strip.go

```diff
@@ -72,6 +72,10 @@ func matchYAMLKey(key string) *regexp.Regexp {
 	return regexp.MustCompile(fmt.Sprintf(`(\s*%s\s*:).+`, key))
 }
 
+func matchYAMLKeyEnding(ending string) *regexp.Regexp {
+	return regexp.MustCompile(fmt.Sprintf(`(\s*(\w|_)*%s\s*:).+`, ending))
```

Comment:
Could you add a small example in a comment explaining the regexp? We got it right now but it gets complicated + `ending` can have multiple interpretations

Reply:
I added a comment, I hope helpful and not confusing...

Reply:
🙇